backend/.profile

```diff
@@ -56,3 +56,6 @@ if [[ "$CF_INSTANCE_INDEX" == 0 ]]; then
     python manage.py collectstatic --noinput &&
     echo 'Finished collectstatic'
 fi
+
+# Make psql usable by scripts, for debugging, etc.
+alias psql='../deps/0/apt/usr/lib/postgresql/*/bin/psql'
```

Comment:
Should this be aliased to an absolute, as opposed to relative path?

Reply:
I believe that is in relation to the `manifest.yml` buildpacks. So if apt_buildpack is listed first, (which we won't be changing) it will be the 0th entity, and the current `postgresql/*` will account for any version.
```bash
vcap@ebfe0af5-cbdf-4b9c-5a56-bc46:/home/vcap/deps$ ls
0  1

vcap@ebfe0af5-cbdf-4b9c-5a56-bc46:/home/vcap/deps$ ls -la 0/lib/postgresql/14/bin/
total 2228
drwxr-xr-x 2 vcap vcap   4096 Aug  9 12:01 .
drwxr-xr-x 4 vcap vcap     28 Aug  9 12:01 ..
-rwxr-xr-x 1 vcap vcap  51952 Aug  9 12:01 clusterdb
-rwxr-xr-x 1 vcap vcap  60280 Aug  9 12:01 createdb
-rwxr-xr-x 1 vcap vcap  52304 Aug  9 12:01 createuser
-rwxr-xr-x 1 vcap vcap  47792 Aug  9 12:01 dropdb
-rwxr-xr-x 1 vcap vcap  47728 Aug  9 12:01 dropuser
-rwxr-xr-x 1 vcap vcap  81464 Aug  9 12:01 pg_amcheck
-rwxr-xr-x 1 vcap vcap 101784 Aug  9 12:01 pg_basebackup
-rwxr-xr-x 1 vcap vcap  39304 Aug  9 12:01 pg_config
-rwxr-xr-x 1 vcap vcap 422240 Aug  9 12:01 pg_dump
-rwxr-xr-x 1 vcap vcap  94048 Aug  9 12:01 pg_dumpall
-rwxr-xr-x 1 vcap vcap  39472 Aug  9 12:01 pg_isready
-rwxr-xr-x 1 vcap vcap  72688 Aug  9 12:01 pg_receivewal
-rwxr-xr-x 1 vcap vcap  52312 Aug  9 12:01 pg_recvlogical
-rwxr-xr-x 1 vcap vcap 155384 Aug  9 12:01 pg_restore
-rwxr-xr-x 1 vcap vcap  88760 Aug  9 12:01 pg_verifybackup
-rwxr-xr-x 1 vcap vcap 716376 Aug  9 12:01 psql
-rwxr-xr-x 1 vcap vcap  64432 Aug  9 12:01 reindexdb
-rwxr-xr-x 1 vcap vcap  68816 Aug  9 12:01 vacuumdb

vcap@ebfe0af5-cbdf-4b9c-5a56-bc46:~$ alias psql='../deps/0/apt/usr/lib/postgresql/*/bin/psql'
vcap@ebfe0af5-cbdf-4b9c-5a56-bc46:~$ psql
psql: error: connection to server on socket "/var/run/postgresql/.s.PGSQL.5432" failed: No such file or directory
        Is the server running locally and accepting connections on that socket?
```

Ill let @mogul speak on this.

Reply:
I think he meant, should it be aliased to an absolute path from the root, which seems pretty reasonable since the intent of the allowed is to remove the need to know exactly where the binary is.